src/pages/Gear/GearItemEditForm.tsx

```diff
@@ -60,6 +81,16 @@ export function GearItemEditForm({ gearItem, closeForm, refreshGear }: Props) {
         <div className="invalid-feedback">This field is required</div>
       </label>
 
+      <div className="mb-2">
+        <label>Location:</label>
+        <Select
+          className="flex-grow-1"
+          isMulti={false}
+          options={optionsGearLocations}
+          onChange={setGearLocation}
```

Comment:
You're missing something like

```suggestion
          onChange={setGearLocation}
          value={optionsGearLocations.find((o) => o.id === location.id)}
```

so that it picks up the initial value